payload-test.json

```diff
@@ -0,0 +1,146 @@
+{
+    "name": "ami_test_cluster",
+    "releaseLabel": "emr-5.28.0",
+    "serviceRole": "emr_service_role",
+    "jobFlowRole": "emr_hbase_ingest",
+    "autoScalingRole": "arn:aws:iam::$ACC_NUMBER:role/AE_EMR_AutoScaling_Role",
+    "hostedZoneId": "$REGION",
+    "customInstanceConfig": {
+        "ec2SubnetId": "$SUBNET_ID",
+        "useSpotPricing": "false",
+        "instanceTemplate": "SMALL",
+        "keepAlivePostJob": "false"
+    },
+    "customEmrConfigs": [
```

Comment:
How come this has a full config in it? I thought we'd agreed to iterate on it!

Reply:
We did, yet CB required content in `"customEmrConfigs":` so I had to give it something.